src/mpid/ch4/src/ch4_impl.h

```diff
@@ -400,6 +400,42 @@ MPL_STATIC_INLINE_PREFIX void MPIDIG_win_hash_clear(MPIR_Win * win)
         }                                                       \
     } while (0)
 
+/* Check both origin|target buffers' size. */
+#define MPIDI_Datatype_check_origin_target_size(_o_datatype, _t_datatype,         \
```

Comment:
Makes sense. I think it might be better for me to add another commit to do this refactoring for all macros in the same file (including the `if { ...` format you mentioned above).

Reply:
👍